### ISSUE TITLE
Benchmark performance gains of scaled endpoint over kuberproxy

### ISSUE THREAD

**Issue by zasherif** [futurewei-cloud/mizar#137]

**Comment by deepak-vij** [comment]
It would be good to compare contrast XDP versus IPVS, IPTables numbers.

**Comment by zasherif** [comment]
Sure we will do that. these numbers are well known now from multiple other projects. The most interesting here is the provisioning time for the scaled endpoint programmed over the set of bouncers vs programming IPTables/IPVs in each host. An of course to quantify the overhead of extra hops (if any)..

**Comment by deepak-vij** [comment]
Yes, it would be good to see runtime performance & provisioning numbers.
